docs/imviz/plugins.rst

```diff
@@ -75,6 +75,10 @@ performant at the cost of accuracy but should be accurate to within a pixel
 for most cases. If approximation fails, WCS linking still automatically
 falls back to full transformation.
 
+For the best experience, it is recommended that you decide what kind of
+link you want and set it at the beginning of your Imviz session,
+rather than later.
+
```

Comment:
can this be removed now?

Reply:
I want to keep it. I think it is a good recommendation regardless.